contrib/babelfishpg_tsql/src/hooks.c

```diff
@@ -2505,6 +2508,186 @@ pltsql_store_func_default_positions(ObjectAddress address, List *parameters, con
 	table_close(bbf_function_ext_rel, RowExclusiveLock);
 }
 
+/*
+ * Update 'funcsignature', 'default_positions', 'flag_validity', 'flag_values', 
+ * 'modify_date', and 'definition' in 'babelfish_function_ext'
+ */
+void
+alter_bbf_function_ext(ObjectAddress address, ObjectWithArgs *owa, List *parameters, const char *queryString, int origname_location)
```

Comment:
Current task is to merge this into the existing `pltsql_store_func_default_positions` function